patches/server/0019-Implement-Paper-VersionChecker.patch

```diff
@@ -77,7 +79,12 @@ index 0000000000000000000000000000000000000000..1a1b50e475b9ede544b2f6d0d36632b2
 +            case -2:
 +                return Component.text("Unknown version", NamedTextColor.YELLOW);
```

Comment:
I was just thinking about maybe adding a notice here as well?
`Unknown version. Please download an official version from papermc.io`
or something like that?

Reply:
Nah, I don't think so. Pretty sure this returns unknown version if you build paper locally and use that. I think unknown version is fine

Reply:
Would servers on old, old paper versions that still use the V1 download API get this message since they can't check? Might be useful for them.